src/components/Posts/EditPostModal.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Dialog,
+  DialogTitle,
+  Button,
+  DialogContent,
+  IconButton,
+  Box,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import { useEffect, useState } from "react";
+import { useAppDispatch, useAppSelector } from "../../state/hooks";
+import { EmojiClickData } from "emoji-picker-react";
+import EmojiPickerIconButton from "../Common/EmojiPickerIconButton";
+import UserAvatar from "../Common/UserAvatar";
+import AddPhotoAlternateOutlinedIcon from "@mui/icons-material/AddPhotoAlternateOutlined";
+import formatTimestamp from "../../utilities/formatTimestamp";
+import axios from "axios";
+import { updatePost } from "../../state/slices/postsSlice";
+
+type EditPostModalProps = {
+  onClose: () => void;
+  openModal: boolean;
+  postId: number;
+  originalPostTextContent: string;
+  originalPostTimeStamp: string;
+  prevEditedPostTimestamp: string;
```

Comment:
Nit, the camel casing is off here. It should be `originalPostTimestamp` and `prevEditedPostTimestamp`. A suggestion too, can we shorten the name to `editedPostTimestamp`?

Reply:
Makes sense. Done